lib/Fmt_ast.ml

```diff
@@ -1789,7 +1789,7 @@ and fmt_expression c ?(box = true) ?pro ?epi ?eol ?parens ?(indent_wrap = 0)
             ?epi const
         $ fmt_atrs )
   | Pexp_constraint
-      ( {pexp_desc= Pexp_pack me; pexp_attributes= []; pexp_loc= _; _}
+      ( {pexp_desc= Pexp_pack me; pexp_attributes= []; pexp_loc; _}
       , { ptyp_desc= Ptyp_package (id, cnstrs)
         ; ptyp_attributes= []
         ; ptyp_loc= _
```

Comment:
I didn't find a case where this location is to be considered.

Reply:
I added some test cases.